local.css

```diff
@@ -126,8 +126,9 @@ table td {
 }
 
 
-.questions {
-	font-style: italic;
+.question {
+    .issue;
```

Comment:
As you see this is retracted PR and I'm planning to send it again due to conflicts, but these lines allow us to extend the inherit `issue` into `question` and change the style of text for it. Usage: `<div class="issue question">foo</div>`.